scripts/caseMapping/CaseMapping.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module CaseMapping where
```

Comment:
I always require an explicit export list. Not sure how Vincent feels though (if he does then an HLint rule could enforce it)

Reply:
Oops, my bad, I made that during the prototyping phase, completely forgot to add it when cleaning...

Reply:
I'm fine either way, since it's a on the side script